tests/integration/common-operator-integ-suite.sh

```diff
@@ -40,6 +41,10 @@ parse_flags() {
         shift
         SKIP_BUILD=true
         ;;
+      --arm64)
```

Comment:
why do we need to pass a parameter? Can't the script figure out the platform itself?

Reply:
I do it but in the specific ocp script here: https://github.com/maistra/istio-operator/blob/365adca0c556d59ec3f330b5b1464a144e717da9/tests/integration/integ-suite-ocp.sh#L23-L28
To have outside the common file those type of validation, but I can added inside of this files if make more sense

Reply:
Done